src/go/rpk/pkg/adminapi/api_partition.go

```diff
@@ -32,10 +32,31 @@ type Partition struct {
 	Replicas    []Replica `json:"replicas"`
 }
 
-// Reconfiguration is the detail of a partition reconfiguration. There are
-// many keys returned, so the raw response is just unmarshalled into an
-// interface.
-type Reconfiguration map[string]interface{}
```

Comment:
@r-vasquez , just wanted to let you know that I've confirmed that I can collect `admin/reconfigurations.json` with `rpk debug bundle` after this change.

Reply:
FWIW an old rpk will miss fields added in a new redpanda, but that might be fine.